README.md

```diff
@@ -1,3 +1,10 @@
+<h1>
+    Danger
+</h1>
+<p>
+    Hard-fork of WalletWasabi for Eagle to play with. No guarantees. If you use it all your coins belong to Eagles.
```

Comment:
hmm the new org name implies something a bit more concrete.

p.s. ;-)
> for the birds
> *slang* Worthless, stupid, or completely undesirable

Reply:
put plural eagles, you can steal coin as well 😅